src/Chirp.Core/ILikeRepository.cs

```diff
@@ -0,0 +1,12 @@
+namespace Chirp.Core;
+
+public interface ILikeRepository
+{
+    public void LikeCheep(int authorId, int cheepId);
+    public void UnlikeCheep(int authorId, int cheepId);
```

Comment:
These could also be done in one action such as if the cheep has been liked, it will be unliked. That is mostly a taste thing, and not a thing you should change if you don't want to.